### DISCUSSION TITLE
sokol.sapp.MouseCursor missing when running ui demos

### DISCUSSION THREAD

**Discussion by smhcpp** [vlang/v#15190] [General]
I am a newbie. I just installed v ui modules and tried to run the demo on arch linux and there was this problem of `sokol.sapp.MouseCursor` not found. I have all the recommended libraries and the only difference might be that I am using a glfw-wayland rather than x11 one. Could this be causing the issue?

So the error line goes as : `/home/mortimer/.vmodules/ui/src/tool_mouse.v:13:25: error: unknown enum 'sokol.sapp.MouseCursor' (type_idx=0)`

**Comment by larpon** [comment]
Try updating both V and V UI by running `v up` and `v update ui`

**Reply by smhcpp** [reply]
I did exactly the same and it worked! Thanks.
